docs/guides/Screen-Nav-Options.md

```diff
@@ -17,21 +17,31 @@ class MyScreen extends React.Component {
 
 **Dynamic Configuration**
 
-Or, each option can be a function that takes the following arguments, and returns the value of the option.
+Or, the options can be a function that takes the following arguments, and returns an object of navigation options that will be override the route-defined and navigator-defined navigationOptions.
 
-- `navigation` - the [navigation prop](/docs/navigators/navigation-prop) for the screen, with the screen's route at `navigation.state`
-- `childRouter` - The child router, if the screen is a navigator
+- `props` - The same props that are available to the screen component
+  - `navigation` - The [navigation prop](/docs/navigators/navigation-prop) for the screen, with the screen's route at `navigation.state`
+  - `screenProps` - The props passing from above the navigator component
+  - `prevOptions` - The default or previous options that would be used if new values are not provided
```

Comment:
We want these arguments to match up with the screen's props. When we render the screen there is no 'previous' options, just the current ones. So that is why I think we should just call it `navigationOptions`